README.md

```diff
@@ -23,6 +23,41 @@ scoop bucket add platformsh https://github.com/platformsh/homebrew-tap.git
 scoop install platform
 ```
 
+### Bash installer
+
+Use the bash installer for an automated installation, using the most preferable way for your system.
+
+```console
+curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
+```
+
+The installer is configurable using the following environment variables:
+
+* `INSTALL_LOG` - the install log file
+* `INSTALL_METHOD` - force a specific installation method, possible values are `brew` and `raw`
+* `INSTALL_DIR` - the installation directory for the `raw` installation method, for example you can use `INSTALL_DIR=$HOME/.local/bin` for a single user installation
+* `VERSION` - the version of the CLI to install, if you need a version other than the latest one
+
+#### Installation configuration examples
+
+<details>
+    <summary>Force the CLI to be installed using the raw method</summary>
+
+    INSTALL_METHOD=raw curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
```

Comment:
Unfortunately this doesn't seem to work in ZSH on macOS?

Variations that do work:

```bash
# inline after pipe
curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | INSTALL_METHOD=raw bash

# export with subshell
(export INSTALL_METHOD=raw; curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash)
```

Reply:
Yes, added the variables before the `bash` command which works in the same way on all operating systems and on ZSH.